test/snippet/search/algorithm/search.cpp

```diff
@@ -0,0 +1,28 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+
+int main()
+{
+    using seqan3::operator""_dna4;
+    std::vector<std::vector<seqan3::dna4>> genomes{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTA"_dna4,
```

Comment:
Please use 
```suggestion
    std::vector<seqan3::dna4_vector> genomes{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTA"_dna4,
```
instead.

Reply:
still open